app/models/post.rb

```diff
@@ -133,83 +134,83 @@ def update_feedback_cache
 
     feedbacks = self.feedbacks.to_a
 
-    self.is_tp = true if feedbacks.index { |f| f.is_positive? }
-    self.is_fp = true if feedbacks.index { |f| f.is_negative? }
-    self.is_naa = true if feedbacks.index { |f| f.is_naa? }
+    self.is_tp = true if feedbacks.index(&:is_positive?)
+    self.is_fp = true if feedbacks.index(&:is_negative?)
+    self.is_naa = true if feedbacks.index(&:is_naa?)
 
-    is_feedback_changed = self.is_tp_changed? || self.is_fp_changed? || self.is_naa_changed?
+    is_feedback_changed = is_tp_changed? || is_fp_changed? || is_naa_changed?
 
     save!
 
-    if self.is_tp && self.is_fp
-      SmokeDetector.send_message_to_charcoal "Conflicting feedback on [#{self.title}](//metasmoke.erwaysoftware.com/post/#{self.id})."
+    if is_tp && is_fp
+      SmokeDetector.send_message_to_charcoal "Conflicting feedback on [#{title}](//metasmoke.erwaysoftware.com/post/#{id})."
     end
 
-    if self.is_fp_changed? && self.is_fp && self.flagged?
-      SmokeDetector.send_message_to_charcoal "**fp on autoflagged post**: #{self.title}](//metasmoke.erwaysoftware.com/post/#{self.id})"
+    if is_fp_changed? && is_fp && flagged?
+      SmokeDetector.send_message_to_charcoal "**fp on autoflagged post**: #{title}](//metasmoke.erwaysoftware.com/post/#{id})"
     end
 
     if is_feedback_changed
-      ActionCable.server.broadcast "topbar", { review: Post.without_feedback.count }
+      ActionCable.server.broadcast 'topbar', review: Post.without_feedback.count
     end
 
-    return is_feedback_changed
+    is_feedback_changed
   end
 
-  def is_question?
-    return self.link.include? "/questions/"
+  def is_question? # rubocop:disable Style/PredicateName
+    link.include? '/questions/'
   end
 
-  def is_answer?
-    return self.link.include? "/a/"
+  def is_answer? # rubocop:disable Style/PredicateName
+    link.include? '/a/'
   end
 
-  def is_deleted?
-    return self.deletion_logs.where(:is_deleted => true).any?
+  def is_deleted? # rubocop:disable Style/PredicateName
+    deletion_logs.where(is_deleted: true).any?
   end
 
   def stack_id
-    return self.link.scan(/(\d*)$/).first.first.to_i
+    link.scan(/(\d*)$/).first.first.to_i
   end
 
   def flagged?
     if flag_logs.loaded?
       flag_logs.select { |f| f.success && f.is_auto }.present?
     else
-      flag_logs.where(:success => true, :is_auto => true).present?
+      flag_logs.where(success: true, is_auto: true).present?
     end
   end
 
   def flaggers
-    User.joins(:flag_logs).where(:flag_logs => {:success => true, :post_id => self.id, :is_auto => true})
+    User.joins(flag_logs).where(flag_logs: { success: true, post_id: id, is_auto: true })
```

Comment:
I thought I went through all of the changes to make sure this got fixed each time it happened, but apparently, I missed a few. Fixed.